wrappers/python/examples/pyglet_pointcloud_viewer.py

```diff
@@ -422,9 +427,17 @@ def run(dt):
     # handle color source or size change
     fmt = convert_fmt(mapped_frame.profile.format())
     global image_data
+
     if (image_data.format, image_data.pitch) != (fmt, color_source.strides[0]):
-        empty = (gl.GLubyte * (w * h * 3))()
-        image_data = pyglet.image.ImageData(w, h, fmt, empty)
+        if state.color:
+            global color_w, color_h
```

Comment:
This global doesn't happen in the `else:` clause

Reply:
I know, I don't need it over there..

Reply:
Oh, I thought it was for image_h... nevermind :)